tensorflow/lite/kernels/bidirectional_sequence_lstm.cc

```diff
@@ -508,12 +508,8 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
   TF_LITE_ENSURE_OK(context,
                     context->ResizeTensor(context, fw_output, fw_output_size));
 
-  // The weights are of consistent type, so it suffices to check one.
-  const bool is_hybrid_op = (fw_input_to_output_weights->type == kTfLiteUInt8 ||
-                             fw_input_to_output_weights->type == kTfLiteInt8);
```

Comment:
It seems this check is different from the others, without checking that input type at the same time. Could you confirm whether this is intended?

Reply:
@haozha111  
L399 this check verification completed, so to keep consistency of method call using utility method but will not affect the existing logic.

`  const TfLiteTensor* input = GetInput(context, node, kInputTensor);
  TF_LITE_ENSURE_EQ(context, input->type, kTfLiteFloat32);
`